sdk/servicebus/Azure.Messaging.ServiceBus/tests/Amqp/AmqpConverterTests.cs

```diff
@@ -127,20 +129,34 @@ public void ReceivedMessageShouldIncrementDeliveryCount()
             Assert.AreEqual(3, sbMessage.DeliveryCount);
         }
 
+        // [Test]
+        // public void CanParseDictionaryValueSection()
```

Comment:
This test started failing when adopting the change from EH to throw in TryGetNetPropertyFromAmqp for the default case statement. I don't think it is really an issue because we are constructing the AmqpMessage by hand and it is not representative of what it would come back as from Service Bus.

Reply:
This is the "throw on unknown" part of the conversion that we were talking about, right?

Reply:
Yes, this test is failing because of that change.